solutions/swb-reference/jest.config.js

```diff
@@ -50,7 +50,7 @@ async function init() {
   };
 
   if (process.env.LAMBDA_URL) {
-    console.log(`LAMBDA_URL = ${process.env.LAMBDA_URL}`);
+    console.log(`LAMBDA_URL = ${process.env.LAMBDA_URL.trim()}`);
```

Comment:
why change the console statement if the below statement no longer matches it?

Reply:
I actually caught that last night and forgot to push the fix i added